app/build.gradle

```diff
@@ -74,7 +74,7 @@ android {
 
 dependencies {
   // Onegini SDK
-  api('com.onegini.mobile.sdk.android:onegini-sdk:11.9.0@aar') {
+  api('com.onegini.mobile.sdk.android:onegini-sdk:11.10.0-SNAPSHOT-local@aar') {
```

Comment:
I guess some testing leftover slipped into the PR.
Also the PR should target develop branch instead of master